content/guides/mobile/index.md

```diff
@@ -0,0 +1,6 @@
+---
+related_pages:
+ - sdks-and-tools
+---
+
+# Mobile
```

Comment:
Definitely needs more content. Ideas:

* Image to show what the SDKs allow a dev to do?
* Maybe an overview of what each SDK enables (and doesn't enable?)

Reply:
Added some text but don't have a great image to use here.